ui/package.json

```diff
@@ -8,15 +8,15 @@
         "node": ">=18.0.0"
     },
     "scripts": {
```

Comment:
[Not about this line] `npm audit` reports `26 vulnerabilities (5 moderate, 20 high, 1 critical)`. It would be nice if the UI team can follow-up updating those. Probably not as part of this PR since it's already big.

Reply:
We are tracking a couple of these currently, but not all of them. Interestingly though, the `1 critical` is not something that has a corresponding CVE ticket in Jira. I do see CVE tickets for other teams and products, but nothing for ACS for some reason.

We have other dependency follow up work we are looking into beyond this PR, so I can add looking into the audit results more closely to our todo list.

Reply:
https://issues.redhat.com/browse/ROX-25751